src/main/java/dukeclasses/ParsedCommand.java

```diff
@@ -0,0 +1,105 @@
+package dukeclasses;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a wrapper class after user input is processed.Created if user input is valid.
+ */
+public class ParsedCommand {
+    private String command;
+    private Integer index;
+    private LocalDate dueDate;
+    private String task;
+
+    /**
+     * Constructor for ParsedCommand for commands that only need String to be instantiated.
+     *
+     * @param command String , usually the command, that is used to instantiated Tasks.
```

Comment:
Should the comma after String appear? Perhaps it could be rephrased. Using 'usually' implies that there is an occasion that command is not used to instantiate Tasks. If so, what is that special cases? I notice the same issue in other places too. Worth thinking about it 😅 
 
```suggestion
     * @param command String the action requested to instantiate Tasks. 
```